pkg/kubelet/server/auth.go

```diff
@@ -99,7 +101,8 @@ func (n nodeAuthorizerAttributesGetter) GetRequestAttributes(u user.Info, r *htt
 			subresources = append(subresources, "pods")
 		case isSubpath(requestPath, healthz.DefaultHealthzPath):
 			subresources = append(subresources, "healthz")
-		case isSubpath(requestPath, configz.DefaultConfigzPath):
+		case isSubpath(requestPath, configz.DefaultConfigzPath),
+			isSubpath(requestPath, flagz.DefaultFlagzPath):
```

Comment:
IIRC, @vinayakankugoyal wanted a separate authz subresource for this? I'm OK with either way. There's a lot of overlap between config file options & flags, so it makes sense to combine these from a security perspective. Not sure which is more expected from a UX perspective.